packages/bpk-component-button/index.js

```diff
@@ -35,3 +41,12 @@ export {
   destructiveThemeAttributes,
 };
 export default BpkButton;
+
+export {
```

Comment:
Not entirely sure this will work as expected. When importing this you will be importing the whole object which I believe will cause webpack to include everything again. 

We probably need to export only one per file. The best solution for me would be adding a `lib` folder that has one export file for each variant, and imports will look like this
```import BpkButtonPrimary from 'bpk-component-button/lib/BpkButtonPrimary'```

Reply:
I would think Webpack's tree shaking can deduce that only `BpkButtonPrimary` is used in thise case

```
import { BpkButtonPrimary } from 'bpk-component-button';
```

That seems to be consistent with what Ross and I observed when trying it in the storybook too

Reply:
👍
If webpack can understand that then all good

Reply:
I suggest we merge this and release an alpha so we can test in a real world scenario

Reply:
Sounds good